core/src/main/scala/org/labrad/Server.scala

```diff
@@ -10,6 +10,7 @@ import org.labrad.concurrent.ExecutionContexts
 import org.labrad.data._
 import org.labrad.errors.LabradException
 import org.labrad.util.{AsyncSemaphore, Logging, Util}
+import org.labrad.util.cli.Command
 import scala.collection.mutable
 import scala.concurrent.{Await, ExecutionContext, Future, Promise}
 import scala.concurrent.duration._
```

Comment:
I think we can probably get rid of `import scala.util.{Success, Failure, Try}` below, unless these imports somehow are linked to the `Future.trySuccess` calls.